src/config/graphical-session/graphics-drivers/optimus.md

```diff
@@ -100,9 +100,8 @@ $ optirun glxinfo | grep "renderer string"
 
 ## Nouveau PRIME
 
-> Note: This method uses the open source `nouveau` driver, which is blacklisted
-> by NVIDIA drivers. Uninstall any NVIDIA driver present on your system and
-> reboot.
+Note: This method uses the open source `nouveau` driver, which is blacklisted by
+NVIDIA drivers. Uninstall any NVIDIA driver present on your system and reboot.
```

Comment:
"Uninstall any proprietary NVIDIA drivers present on your system and reboot. The open source `nouveau` drivers are incompatible with proprietary NVIDIA drivers.

Reply:
Hmm, is this so? i genuinely don't know myself, but https://github.com/void-linux/void-docs/pull/215/files suggests it might not be.

Looking at the current version of `optimus.md`, there are other notes in that page that need to be reviewed (e.g. under the "Bumblebee" heading), and the page needs to be reviewed more generally (e.g. it should be "Nvidia", not "NVIDIA").

Reply:
I have no idea if it's true. I've only used nouveau in the past, and don't have any nvidia hardware right now I can actually test this with. Maybe it would be best to do nothing further here, and this page can be (hopefully) reviewed by someone who knows how nvidia works.

Reply:
@st3r4g: Any thoughts on this?
@jeffayle: Could you please take a look at the other notes in `optimus.md`, and make changes to this PR as appropriate?

Reply:
You can't have both driver running on the system, so in this sense they're incompatible. The easy way is to just uninstall nvidia, which is what I expect most users will do. You can also keep nvidia installed and adjust things manually as described in the nvidia PR, but that requires extra steps and I don't really see strong reasons to do it.

@flexibeast NVIDIA's name is always all caps in their official README, and also in other docs like the Arch wiki. If that's the official way to write their name, why should we change it to Nvidia?

Reply:
@st3r4g: You're right. i thought i'd seen official stuff where it was written as 'Nvidia', but i must have been wrong.

Reply:
what's the final word on this? Should I go with the wording I originally proposed, is this something that is being addressed in another pr or open issue?

Reply:
@jeffayle you could link to this header https://github.com/void-linux/void-docs/blob/master/src/config/graphical-session/graphics-drivers/nvidia.md#reverting-from-nvidia-to-nouveau